sources/packages/backend/libs/services/src/sfas/sfas-application.service.ts

```diff
@@ -20,30 +20,23 @@ export class SFASApplicationService extends DataModelService<SFASApplication> {
 
   /**
    * Validates before an application submission to see if there is an overlapping SFAS fulltime application existing.
-   * @param sin
-   * @param birthDate
-   * @param lastName
-   * @param studyStartDate
-   * @param studyEndDate
+   * @param studentId student id.
+   * @param studyStartDate study start date.
+   * @param studyEndDate study end date.
    * @returns SFAS fulltime application.
    */
   async validateDateOverlap(
-    sin: string,
-    birthDate: string,
-    lastName: string,
+    studentId: number,
     studyStartDate: string,
     studyEndDate: string,
   ): Promise<SFASApplication> {
     return this.repo
       .createQueryBuilder("sfasApplication")
       .select(["sfasApplication.id"])
-      .innerJoin("sfasApplication.individual", "sfasFTstudent")
+      .innerJoin("sfasApplication.individual", "sfasIndividual")
+      .innerJoin("sfasIndividual.student", "student")
```

Comment:
In some situations TypeOrm allow the id directly in the `where` without the need for a `innerJoin`, as shown below. Can you test it please?
```ts
return this.repo
      .createQueryBuilder("sfasApplication")
      .select(["sfasApplication.id"])
      .innerJoin("sfasApplication.individual", "sfasIndividual")
      .where("sfasApplication.applicationCancelDate IS NULL")
      .andWhere("sfasIndividual.student.id = :studentId", { studentId })
```